website/docs/docs/dbt-versions/experimental-features.md

```diff
@@ -21,3 +21,7 @@ To enable or disable experimental features:
 2. Find Experimental features at the bottom of Your Profile page.
 3. Click **Beta** to toggle the features on or off as shown in the following image.
    ![Experimental features](/img/docs/dbt-versions/experimental-feats.png)
+
+## Beta terms and conditions
+
+By using or enabling features that are not yet in general release ("Beta Features"), you agree to the [Beta Features Terms and Conditions](/assets/beta-tc.pdf).
```

Comment:
```suggestion
By using or enabling features that are not yet in general release (beta features), you agree to the [Beta Features Terms and Conditions](/assets/beta-tc.pdf).
```

Reply:
nice one, thanks @nghi-ly ! i do agree with you here but I'm going to stick with "Beta Features" because that's how it's presented legally in the pdf so I'd like to be consistent there. 

I've accepted your other suggestion though so let me know if you have any strong feelings on that!

Reply:
@mirnawong1 : sounds good to me! 👍